tests/release/pipelines/fbc_release.go

```diff
@@ -309,7 +361,14 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 		},
 	})
 	Expect(err).NotTo(HaveOccurred())
+/*
```

Comment:
is this meant to be kept in here?

Reply:
No, I forgot to change it. Updated. Thanks